repose-aggregator/docs/src/asciibinder/welcome/release-notes.adoc

```diff
@@ -20,36 +39,20 @@
-* https://repose.atlassian.net/browse/REP-7486[REP-7486] - Documented removal and replacement solution for the Response Messaging Service.
-* https://repose.atlassian.net/browse/REP-7490[REP-7490] - Cleaning up some log messages in pursuit of more meaningful logging.
+* *Breaking Change* | *Updated* | https://repose.atlassian.net/browse/REP-7490[REP-7490] - Cleaning up some log messages in pursuit of more meaningful logging.
 ** Changing the `org.eclipse.jetty` logger level from `off` to `warn` in the default `log4j2.xml` configuration file.
 ** Removing a log event with a message including `Repose Devs might care about this trace` by the `org.openrepose.commons.config.parser.jaxb.JaxbConfigurationParser` logger.
 ** Changing the log level of events relating to unpacking artifacts by the `org.openrepose.commons.utils.classloader.EarClassProvider` logger from `debug` to `trace`.
-* https://repose.atlassian.net/browse/REP-7519[REP-7519] - Added support for proxies to the <<../services/http-client.adoc#, HTTP Client Service>>.
+* *Breaking Change* | *Removed* | https://repose.atlassian.net/browse/REP-4992[REP-4992] - Removed ambiguous setting for chunked encoding options on <<../services/http-client.adoc, HTTP Client/Connection Pool Service>>.
+* *Breaking Change* | *Removed* | https://repose.atlassian.net/browse/REP-7201[REP-7201] - The Rackspace Auth User, SAML Policy Translation, and Attribute Mapping Policy Validation filters were removed and placed in their own bundle.
+* *Breaking Change* | *Removed* | https://repose.atlassian.net/browse/REP-5326[REP-5326] - Removed the deprecated via attribute from the <<../architecture/container.adoc#, Container configuration>>.
+* *Breaking Change* | *Removed* | https://repose.atlassian.net/browse/REP-7338[REP-7338] - The WAR deployment was removed as an option.
+* *Breaking Change* | *Removed* | https://repose.atlassian.net/browse/REP-4990[REP-4990] - Removed deprecated cache attributes from the <<../filters/openstack-identity-v3.adoc#, OpenStack Identity v3 Filter>>.
+** Cache timeouts are now defined in seconds rather than milliseconds.
+* *Breaking Change* | *Removed* | https://repose.atlassian.net/browse/REP-7428[REP-7428] - Removed support for URI-based tenant validation in the <<../filters/keystone-v2-authorization.adoc#, Keystone v2 Authorization Filter>> and <<../filters/keystone-v2.adoc#, Keystone v2 Filter>>.
+* *Breaking Change* | *Removed* | https://repose.atlassian.net/browse/REP-4993[REP-4993] - Flush output filter was removed.
+* *Breaking Change* | *Removed* | https://repose.atlassian.net/browse/REP-7314[REP-7314] - Removed support for <<../architecture/system-model.adoc#,system model>> clusters.
+* *Breaking Change* | *Removed* | https://repose.atlassian.net/browse/REP-7486[REP-7486] - Removed the Response Messaging Service itself and documented a replacement solution.
```

Comment:
I updated the release notes with categories. These should be the 3 primary and 3 secondary categories that we settled on. Primary categories are:

- (Intentionally empty, the "normal" category)
- *Bug Fix*
- *Breaking Change*

Secondary categories are:
- *Added*
- *Updated*
- *Removed*

I chose to make the impact statement the primary category since I anticipate a user will gravitate towards that first. That is, the impact statement answers the question, "do I need to update something in my deployment?" The diff statement is the secondary category and answers the fundamental question on change, "how did this change?"

Categories are grouped together with primary category grouping prioritized over secondary category grouping.

I used words for the categories for now. This was mostly just to get the grouping in place. We can transition to symbols and/or colors as desired. I believe the goal is to get some images together to replace the text. Those images may map colors to the primary category and symbols to the secondary category, or they may do something else entirely. Alt text will be used in conjunction with the images in consideration of accessibility and internationalization.